lib/org/openpsa/user/accounthelper.php

```diff
@@ -320,6 +330,25 @@ public function check_password_age() : bool
         return false;
     }
 
+    /**
+     * Function sends an email to the user with username and password
+     *
+     * @return boolean indicates success
+     */
+    public function welcome_email(midcom_db_person $person) : bool
```

Comment:
Sorry, what I meant was: The `$person` parameter seems unnecessary, because we already get a person in the constructor. Also, the current version is inconsistent, because you take username and email from `$person`, but `$this->set_account` modifies `$this->person`